etc/p3-config/p3-config.yml

```diff
@@ -787,6 +819,7 @@ p3_image_mlnx_ofed:
     - "mlnx-ofed"
     - "selinux-permissive"
     - "alaska-extras"
+    - "openssh-server"
```

Comment:
Do we still explicitly need ``openssh-server`` now that we are using ``centos`` element?